presets/4.4/tune/uav_tech/UAV_tech_5in_Freestyle_575-650.txt

```diff
@@ -0,0 +1,129 @@
+#$ TITLE: UAV Tech - 5" Freestyle withOUT GoPro (575-650g AUW)
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
```

Comment:
so new presets file should be marked as EXPERIMENTAL for safety.
Exceptions are some small trivial presets.

Reply:
i just renamed the file for the 5" preset that has been in use for a while.